### DISCUSSION TITLE
Projects (beta) - allow drag & drop reordering of views

### DISCUSSION THREAD

**Discussion by felixfbecker** [community/community#5496] [Projects and Issues]
I'd really like to be able to control the order of views in the project in the tab bar like in Google sheets, so that I can order it like "Current iteration", "Next iteration", "Backlog" etc and add a view "in between" or at the start without having to delete all views and start again. Currently it seems to be based on creation order.

**Comment by zepfietje** [comment]
This is a duplicate of https://github.com/github/feedback/discussions/4803.
Would be great if you could upvote that discussion.
